packages/sources/src/components/header/header.models.ts

```diff
@@ -7,6 +7,7 @@ export interface Header {
     url: string;
     active?: boolean;
   }[];
+  noMainMenu: boolean;
```

Comment:
Geen halszaak maar wil het toch benoemd hebben: De component models zijn het contract tussen de functionaliteit en de techniek.

Dit component kent geen functionaliteit "no main menu": De functionaliteit is dat het _zonder menu moet werken_, ofwel geen `mainMenu` of een lege lijst.

`noMainMenu` is dan ook geen eigenschap in het Stencil component (wat goed is) maar je hebt 't alleen nodig om in de templates een lege lijst of `Header.mainMenu` mee te geven.

Ik heb die keuze een laag eerder verplaatst, naar de argsMapper. Dat is de plek waar de args (en eventueel content uit `.content.ts` files) worden gemapt naar het model:

```
export function headerArgsMapper(a: HeaderArgs): Header {
  return {
    logo: a.logo,
    subLogo: a.subLogo,
    showSubLogo: a.showSubLogo,
    mainMenu: a.noMainMenu ? undefined : a.mainMenu,
[...]
```

Nou moet ik deze branch toch rebasen dus ik heb die 3 changes doorgevoerd.

Verder mooi werk weer